server/src/stateManagement.mjs

```diff
@@ -738,6 +742,7 @@ export const testExports={
 }
 export {
   state,
+  defaultUserArray,
```

Comment:
We can just call userManagement.getUsers(). We don't need to keep track of "default" users